docs/DMS/PRIMARY-DATA-STORAGE/README.md

```diff
@@ -278,6 +166,94 @@ for a demo on experimenting with partitioning to find a good size. This will be
 where we compute the `partition_key` from an appropriate GUID in the table to allow for partition elimination
 on queries and partition-aligned indexing.
 
+#### Query handling
+
+See [Queries Using the Relational Database](./RELATIONAL-QUERIES.md).
+
+### Planning ahead for performance considerations
+
+- The primary `Documents` table will use a `bigint` (64 bit integer) for the primary key.
+- Out of the box, the `Documents` will be split into 16 different partitions.
+  These partitions are in different files, thus reducing contention compared
+  to a single partition in one gigantic file.
+- A partition key column, based on the primary key, will improve indexing and
+  will help distribute all of the stored records evenly across the available
+  partitions.
+- An implementation must be able to modify to more or fewer partitions as
+  desired. In this design, the partition key limits the potential number of
+  partitions to 256.
+
+The development team has [executed initial
+experiments](../../../POC-Applications/POC.Databases/) to compare some aspects
+of performance between this structure and the ODS database from the ODS/API
+Platform v7. Similar records were inserted into three tables: `Student`,
+`StudentSchoolAssociation`, and `StudentSectionAssociation`. Running a virtual
+machine with SQL Server 2022 and Windows Defender disabled, large numbers of
+inserts were run and timed for direct comparison of the insert performance. The
+following tables summarize the initial findings with 1 million records:
+
+| Database | Insert Time | Storage (KB) |
+| -------- | ----------- | ------------ |
+| DMS      | 02:49:42    | 571,528      |
+| ODS      | 03:22:34    | 1,268,968    |
+
+The techniques used in these experiments do not provide perfect comparisons, but
+we believe these results are "essentially" comparable. And that comparison is
+very favorable, supporting the hypothesis of high performance with this database
+design.
+
+## Implementation
+
+The proof-of-concept process helped uncover some details that need to guide the
+implementation:
+
+- Compared to `varbinary`, using `varchar` for the JSON documents results in
+  higher performance and smaller storage requirements.
```

Comment:
@stephenfuqua 
'varchar' requires more storage space compared to 'varbinary'.

Reply:
```suggestion
  higher performance, albeit at the cost of approx 20% higher disk space usage.
```